File: gp-web/nlp-interface/src/main/java/com/scarike/gp/web/grpc/Keyword.java
package com.scarike.gp.web.grpc;


import lombok.Data;

@Data
public class Keyword {
    private QueryMethod type;
    private String center;
    private Integer distance;
    private Integer poiCode;
    private String key;

    public enum QueryMethod {
        POINT, LINE
    }
}

File: gp-web/nlp/src/keyword.py
class Keyword():
    def __init__(self) -> None:
        self.type=QueryType.UNDEFINED
        self.center=""
        self.distance=0
        self.poiCode=0

class QueryType():
    POINT = 1
    LINE = 2
    UNDEFINED = 3

class Clazz():
    def __init__(self,type:QueryType,distance_type:bool) -> None:
        self.query_type=type
        self.is_accurate=distance_type